doc/classes/SpatialMaterial.xml

```diff
@@ -639,5 +644,14 @@
 		<constant name="DISTANCE_FADE_OBJECT_DITHER" value="3" enum="DistanceFadeMode">
 			Smoothly fades the object out based on the object's distance from the camera using a dither approach. Dithering discards pixels based on a set pattern to smoothly fade without enabling transparency. On certain hardware this can be faster than [constant DISTANCE_FADE_PIXEL_ALPHA].
 		</constant>
+		<constant name="FALLBACK_MODE_NONE" value="0" enum="FallbackMode">
+			This material won't have a fallback. The application will stop to compile the real conditioned shader when it's used for the first time.
```

Comment:
```suggestion
			This material won't have a fallback. The project will stop to compile the real conditioned shader when it's used for the first time. This can lead to the project temporarily freezing during gameplay.
```

Reply:
Done!